targets/newTarget/source/exampleSourceFile.txt

```diff
@@ -6,13 +6,3 @@ You can have as many files and subdirectories as you want, and they're all copie
 
 EJS tags are ignored, so, <%- thisChunk %> is not evaluated in any way.
 (If it were, it would probably throw an exception about how "thisChunk" is not a known variable)
-
-
-
-
-***********************************************************************
-If you can see this, then the SDKGenerator is syncing the ghtest branch
```

Comment:
just confirming- "ghtest" is not a typo?

Reply:
ghtest was deliberate at the time.
It was short for "Git Hub [Gated checkin] Test"
This was a temporary change that was accidentally committed.
I only just noticed it for the first time.
It's not technically part of this feature, but trivially difficult, and minorly pointless, to separate to its own PR.